JHUGenerator/mod_Misc.F90

```diff
@@ -1542,14 +1542,15 @@ SUBROUTINE PrintLogo(TheUnit, title)
     write(TheUnit, *) " ", CenterWithStars("Spin and parity determination of single-produced resonances at hadron colliders", linelength)
     write(TheUnit, *) " ", CenterWithStars("", linelength)
     write(TheUnit, *) " ", CenterWithStars("I. Anderson, S. Bolognesi, F. Caola, J. Davis, Y. Gao, A. Gritsan,", linelength)
-    write(TheUnit, *) " ", CenterWithStars("Z. Guo, C. Martin, K. Melnikov, R. Rontsch, H. Roskes, U. Sarica,", linelength)
-    write(TheUnit, *) " ", CenterWithStars("M. Schulze, N. Tran, A. Whitbeck, M. Xiao, Y. Zhou", linelength)
+    write(TheUnit, *) " ", CenterWithStars("L. S. Mandacaru Guerra, Z. Guo, C. Martin, T. Martini, K. Melnikov, R. Pan,", linelength)
```

Comment:
Why not L. S. M. Guerra? Is Mandacaru Guerra the full last name?

Reply:
Not the right place to discuss last names, those are already in the publication, could be checked there

Reply:
It is ok if we would like to stay consistent with what is on the publications. Then I suggest the following changes to be applied:
A. Gritsan -> A. V. Gritsan
H. Roskes -> J. Roskes
N. Tran -> N. V. Tran
Would that make sense? The same change needs to be propagated to MELA of course.

Reply:
I don't specifically care what we put with respect to my name, but with regard to Lucas: his correct last name is Mandacaru Guerra, not Guerra.  It's used for example his JHU ID and email (which I don't want to post on github where anyone can see it).  This is common for people from Spanish and Portuguese speaking places.